js/ui/data_grid/ui.data_grid.export.js

```diff
@@ -825,18 +823,33 @@ dataGridCore.registerModule("export", {
 
                 },
 
-                _correctSearchPanelPosition: function(items) {
-                    items.sort(function(itemA, itemB) {
-                        var result = 0;
+                _correctItemsPosition: function(items, exportItems) {
```

Comment:
With the PR changes, this function not only **correct** items position but add the export items. When someone else will work with it, he will need to spend more time to understand what this function actually does.